.github/workflows/maven.yml

```diff
@@ -26,5 +26,10 @@ jobs:
     name: Verify
     uses: apache/maven-gh-actions-shared/.github/workflows/maven-verify.yml@v3
     with:
+      jdk-matrix: '[ "20", "17", "8" ]'
+      ff-jdk: '20'
```

Comment:
next LTS will be 21. Should we really care of the intermediary 20?

Reply:
21 is not yet available and unfortunately, we are not testing all our projects with Java 20 yet

Reply:
ok Java 20 removed - save CI